src/test/fuzz/bip324.cpp

```diff
@@ -98,7 +98,7 @@ FUZZ_TARGET(bip324_cipher_roundtrip, .init=Initialize)
             unsigned damage_bit = provider.ConsumeIntegralInRange<unsigned>(0,
                 (ciphertext.size() + aad.size()) * 8U - 1U);
             unsigned damage_pos = damage_bit >> 3;
-            std::byte damage_val{(uint8_t)(1U << (damage_bit & 3))};
+            std::byte damage_val{(uint8_t)(1U << (damage_bit & 7))};
```

Comment:
Good point, I agree. In a similar sense, the damage position should probably be calculated as `damage_bit / 8` instead of a right-shift by 3. I assume the original motivation for using "bit-fiddling" instead was performance (worries that the modulus operation is significantly slower), but it seems both variants emit the same code on recent compilers, even without optimization turned on: https://godbolt.org/z/EjY59a36s

Will leave the PR as it is now, as it has already two ACKs, but will consider working on a follow-up PR that replaces more constructs like this (i.e. division and bitwise AND with a powers-of-two constant) in our code-base.